src/main/java/dev/jbang/util/Util.java

```diff
@@ -467,7 +467,7 @@ static public void errorMsg(String msg, Throwable e) {
 			if (msg != null) {
 				infoMsg(e.getMessage());
 			}
-			infoMsg("Run with --verbose for more details");
+			infoMsg("Run with --verbose for more details. The --verbose must be placed before the jbang command. I.e. jbang --verbose run [...]");
```

Comment:
Just to be more explicit, you can basically put it anywhere, _except_ for the `run` , `alias add` and `app install` commands where you can't put at after the main argument because in those 3 cases it would be considered a user parameter, as Max explained.